cmd/kyma/provision/gardener/cmd.go

```diff
@@ -49,7 +49,7 @@ Use the following instructions to create a service account for a selected provid
 	cmd.Flags().StringVarP(&o.Secret, "secret", "s", "", "Name of the Gardener secret used to access the target provider. (required)")
 	cmd.Flags().StringVarP(&o.KubernetesVersion, "kube-version", "k", "1.15.4", "Kubernetes version of the cluster.")
 	cmd.Flags().StringVarP(&o.Region, "region", "r", "europe-west3", "Region of the cluster.")
-	cmd.Flags().StringVarP(&o.Zone, "zone", "z", "europe-west3-a", "Zone of the cluster.")
+	cmd.Flags().StringSliceVarP(&o.Zone, "zone", "z", []string{"europe-west3-a"}, "Zone of the cluster. eg. --zone=\"europe-west3-a,europe-west3-b\"")
```

Comment:
Description states it's the zone of the cluster, but the example has a list of zones. Is it a list of possible zones for the cluster? Is the first one prioritized? Please add more to the description